lib/core/rancher/step.sh

```diff
@@ -40,6 +57,8 @@ function ih::setup::core.rancher::install() {
     brew list ih-rancher > /dev/null 2>&1
     IHRANCHERINSTALLED=$?
 
+    echo " Rancher installed :" $RANCHERINSTALLED
+    echo " IH Rancher installed :" $RANCHERINSTALLED
```

Comment:
Did you mean `$IHRANCHERINSTALLED`?

Underscores make `ALLCAPS` `EASIER_TO_READ`.

You can (and probably should) write this as 
```
echo " IH Rancher installed : $RANCHERINSTALLED "
```

*BUT* why are you `echo`ing this at all?

Reply:
My mistake. I forgot to delete this when I was testing.